site/en/tutorials/text/text_classification_rnn.ipynb

```diff
@@ -363,21 +364,21 @@
         "\n",
         "1. This model can be build as a `tf.keras.Sequential`.\n",
         "\n",
-        "1. The first layer is the `encoder`, which converts the text to a sequence of token indices.\n",
+        "2. The first layer is the `encoder`, which converts the text to a sequence of token indices.\n",
         "\n",
-        "2. After the encoder is an embedding layer. An embedding layer stores one vector per word. When called, it converts the sequences of word indices to sequences of vectors. These vectors are trainable. After training (on enough data), words with similar meanings often have similar vectors.\n",
+        "3. After the encoder is an embedding layer. An embedding layer stores one vector per word. When called, it converts the sequences of word indices to sequences of vectors. These vectors are trainable. After training (on enough data), words with similar meanings often have similar vectors.\n",
         "\n",
         "  This index-lookup is much more efficient than the equivalent operation of passing a one-hot encoded vector through a `tf.keras.layers.Dense` layer.\n",
         "\n",
-        "3. A recurrent neural network (RNN) processes sequence input by iterating through the elements. RNNs pass the outputs from one timestep to their input on the next timestep.\n",
+        "4. A recurrent neural network (RNN) processes sequence input by iterating through the elements. RNNs pass the outputs from one timestep to their input on the next timestep.\n",
         "\n",
         "  The `tf.keras.layers.Bidirectional` wrapper can also be used with an RNN layer. This propagates the input forward and backwards through the RNN layer and then concatenates the final output. \n",
         "\n",
-        "  * The main advantage to a bidirectional RNN is that the signal from the beginning of the input doesn't need to be processed all the way through every timestep to affect the output.  \n",
+        "  * The main advantage of a bidirectional RNN is that the signal from the beginning of the input doesn't need to be processed all the way through every timestep to affect the output.  \n",
         "\n",
         "  * The main disadvantage of a bidirectional RNN is that you can't efficiently stream predictions as words are being added to the end.\n",
         "\n",
-        "1. After the RNN has converted the sequence to a single vector the two `layers.Dense` do some final processing, and convert from this vector representation to a single logit as the classification output. \n"
+        "5. After the RNN has converted the sequence to a single vector of the two `layers.Dense` do some final processing, and convert from this vector representation to a single logit as the classification output. \n"
```

Comment:
This `of` changes the intended meaning.

The RNN converts the sequence to a vector. The two Dense layers process that vector.

Reply:
Of course, I will return it as before.
P.S: maybe comma should be there...